### DISCUSSION TITLE
Catwalk and fance

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4144] [Feature Suggestions]
catwalk without a fence. And the fence itself without a catwalk. Later there will be screenshots if you do not understand

**Comment by SimpleFoxOfficial** [comment]
yep this will be cool

**Comment by FrozenDragon0** [comment]
Already been suggested by me years ago: https://github.com/BEEmod/BEE2-items/issues/1274
